README.md

```diff
@@ -41,6 +42,22 @@ Use in your template.
 ```
 
 Where `@page` is a `%Scrivener.Page{}` struct returned from `Repo.paginate/2`.
+This means you need to add some code in your controller like:
+
+```elixir
+page = MyApp.User
+  |> where([p], p.age > 30)
+  |> order_by([p], desc: p.age)
+  |> MyApp.Repo.paginate(params)
```

Comment:
Lets leave out the extra query conditions and define what `params` are so they'll know what to expect to pass in. Additionally, we should use good style in the example and align the pipes. Something like this is what I'm thinking:

```elixir
#                        params = %{"page" => _page}
def index(conn, params) do
  users = MyApp.User
               # Other query conditions can be done here
               |> MyApp.Repo.paginate(params)
  render conn, :index, users: users
end
```